next-auth.d.ts

```diff
@@ -1,6 +1,4 @@
-import NextAuth, { DefaultSession } from 'next-auth';
-import { Session } from 'next-auth';
-import { JWT } from 'next-auth/jwt';
+import { DefaultSession } from 'next-auth';
 
 declare module 'next-auth' {
```

Comment:
Should `followedTags`, `lastLogin`, and `emailPreference` be added to the `Session` interface here since we add those values?

Reply:
@mcolella14 @harikakotipalli if we do this then we can likely just use the `Session` interface instead of having to create the `User` interface too right?

Reply:
@justinmckenzie Not all fields in user need to be in session - what if there is some client side call eg: home page content , which deals with new field which needs to be in User interface , but not needed in session.

Reply:
@mcolella14 : +1 , adding those fields to session. If I don't add I don't think we can access them in useSession, getSession, <SessionProvider> etc.